test/test-track.sh

```diff
@@ -292,3 +292,136 @@ begin_test "track blocklisted files with glob"
   grep "Pattern .git\* matches forbidden file" track.log
 )
 end_test
+
+begin_test "track lockable"
+(
+  set -e
+
+  repo="track_lockable"
+  mkdir "$repo"
+  cd "$repo"
+  git init
+
+  # track *.jpg once, lockable
+  git lfs track --lockable "*.jpg" | grep "Tracking \*.jpg"
+  numjpg=$(grep -e "\*.jpg.*lockable" .gitattributes | wc -l)
```

Comment:
> `grep -c` should be used in favor of `grep | wc -l` for clarity.

Problem with that is that `grep -c` returns a failure code when there are no results (which we validly test for in many cases), which means you lose the gained clarity trying to ignore that. I think `wc -l` is clearer while still dealing with `0` result cases.